### DISCUSSION TITLE
开源软件供应链之道 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#120] [Announcements]
# 开源软件供应链之道 - 「开源之道」

https://opensourceway.community/posts/supply_chain/try-to-explain-open-source-supply-chain/

**Comment by ZoomQuiet** [comment]
看起来说了很多, 可其实又什么都没说;
自由开源软件之所以难以立即平替,
是因为人家包含了大量义务工程师长期海量工作时间在内, 
已经形成了知识壁垒,
但是, 国家或是厂商, 是完全可以不顾许可, 拿最新版本封闭起来发行自己的,
北韩, 就是这么干的;

而从以往笔者的言论中是倡导一个前提,
互联网中, FLOSS 项目是无法被阻止的,
所以, 最简单的供应链接姿势就是加入其中;
可现实是, 互联网是可以被控制的,
我们或是人家, 都可以一键关闭一个国家或是地区所有互联网连接,
将这个地区以往开发者的贡献锁在别人的仓库中,
也就是说, 软件供应链的代码逻辑链是难以打破的,
但是, 供应链的物理通路是可控制的;

问题来了, 这时, 小国只能怂了, 大国呢?

既然, 普通人不关心手中的设备运行什么样的代码,
那么, 谁应该为此负责?

|> 240725 晨糟:
2/4(每天认真吐糟不得超过4次)
